validation/misc_props.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/mndrix/tap-go"
+	rspecs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/runtime-tools/specerror"
+	"github.com/opencontainers/runtime-tools/validation/util"
+	uuid "github.com/satori/go.uuid"
+)
+
+func saveConfig(path string, v interface{}) error {
+	data, err := json.Marshal(v)
+	if err != nil {
+		return err
+	}
+
+	return ioutil.WriteFile(path, data, 0644)
+}
+
+func main() {
+	t := tap.New()
+	t.Header(0)
+	bundleDir, err := util.PrepareBundle()
+	if err != nil {
+		util.Fatal(err)
+	}
+	defer os.RemoveAll(bundleDir)
+	configFile := filepath.Join(bundleDir, "config.json")
+
+	type extendedSpec struct {
+		rspecs.Spec
+		Unknown string `json:"unknown,omitempty"`
```

Comment:
I don't quite understand what that does.

Reply:
I use this to test ’runtimes that are reading or processing this configuration file MUST NOT generate an error if they encounter an unknown property‘.  'Unknown' here (in my commit) means the 'unknown property', so when a runtime reads this, it does not know what it means and it should not generete an error.